### ISSUE TITLE
React360-Redux:Communication between 2 roots components via redux

### ISSUE THREAD

**Issue by xristlxcv** [facebook/react-360#714]
i want to create a vr shop and i have 2 panels.On the center panel i have the produducts and on the right panel i want to show information.I have a store which is updated but it doesn't pass information in second panel when i click at my products.The second panel appears continously the initial state which has the store.
![second](https://user-images.githubusercontent.com/51959400/59690539-422b0080-91ea-11e9-9fe3-1f180f34a55a.png)

**Comment by andrewimm** [comment]
Have you looked at this example: https://facebook.github.io/react-360/docs/example-multisurface.html

It does exactly what you describe, with a Redux-like demo library

**Comment by xristlxcv** [comment]
yes...but because i'am beginer i didn't understand by that method and i try to make something similar with redux .
Βut I can not understand why my data in the second panel is not updated with my redux method
